src/ui/SelectionObserver.ts

```diff
@@ -138,6 +138,8 @@ export  class SelectionObserver {
         selection: $selection,
       };
     });
-    changed && callback && callback(this.takeRecords(), this);
+    if (changed && callback) {
```

Comment:
callback can be removed from if you use null safe call `callback?.(this.takeRecords(), this);`